LICENSE

```diff
@@ -210,4 +210,38 @@ for text of these licenses.
 Apache License 2.0
 --------------------------------------
 
-common/src/main/java/org/apache/celeborn/common/network/util/LimitedInputStream.java
```

Comment:
Why do we remove this line? This is required because the `LimitedInputStream` class stems from Guava and is still bundled in our source package.

Please be cautious to remove any line in the LICENSE file.

Reply:
it is moved to line 236

Reply:
@carp84, `LimitedInputStream` uses the Guava third party and I have moved to line 236. Does not remove this line. PTAL.

Reply:
ok, missed since there're quite some comments in between. Thanks for the clarification guys.